js/main.js

```diff
@@ -21,7 +21,103 @@ function submitEntry(event) {
   data.nextEntryId++;
   photo.setAttribute('src', photoDefault);
   entryLog.reset();
-
+  var newSubmit = renderJournalEntry(entryData);
+  ul.prepend(newSubmit);
 }
 
 entryLog.addEventListener('submit', submitEntry);
+
+// var mainForm = document.querySelector('#main-form');
+
+/*
+<li>
+  <div class="container"> x
+    <div class="row"> x
+      <div class="column-half"> x
+        <img class="photo" src="images/placeholder-image-square.jpg"> x
+      </div>
+      <div class="column-half">
+        <div>
+          <h2>Title</h2>
+        </div>
+        <p>
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Possimus suscipit quidem animi iure ipsam. Facere maiores, rerum autem nesciunt voluptates tempora aut consectetur, iure sed aliquid ut? Nesciunt, voluptatibus beatae?
+        </p>
+      </div>
+    </div>
+  </div>
+</li> */
+
+function renderJournalEntry(entry) {
+  var li = document.createElement('li');
+
+  var container = document.createElement('div');
+  container.setAttribute('class', 'container');
```

Comment:
Why an individual `container` for each `li`? That seems unnecessary. You usually only need one `container` at the most external layer of a section. For this project, I usually recommend making the `main` element the `container`

Reply:
Fixed